llvm/utils/lit/tests/Inputs/per-test-coverage/per-test-coverage.py

```diff
@@ -1,12 +1,11 @@
-# Check that the environment variable is set correctly
-# RUN: %{python} %s | FileCheck -DINDEX=1 %s
-# RUN: %{python} %s | FileCheck -DINDEX=2 %s
+#Check that the environment variable is set correctly
```

Comment:
This file shouldn’t be modified at all. Please make it match what is on main.

Reply:
yes I will change that right now